packages/gatsby-plugin-sass/src/gatsby-node.js

```diff
@@ -29,7 +33,7 @@ exports.onCreateWebpackConfig = (
       ? [loaders.null()]
       : [
           loaders.miniCssExtract(),
-          loaders.css({ ...cssLoaderOptions, importLoaders: 2 }),
+          loaders.css({ importLoaders: 2, ...cssLoaderOptions }),
```

Comment:
why the order change?  this make it possible for users to change `importLoader` - I'm not sure if the order wasn't very intentional before (to not allow it) because it might be footgun situation (but I don't remember specifics of `importLoaders` and its impact)

Reply:
This fixes https://github.com/gatsbyjs/gatsby/issues/28278 & via git blame I couldn't find an explanation to that order so assumed it was just arbitrarily chosen

Reply:
I mean - this will be new major so behaviour change from this swap is acceptable - but maybe we should add a note about it  in "changelog" part of readme? I don't see it being mentioned in docs (some users might have `importLoaders` in their config already, it just doesn't do anything right now and after upgrade it would suddenly have an effect)